### PR TITLE
Add summary to step run

### PR DESCRIPTION
**Pull request by lengau**
This writes a brief summary (one line per environment) to the GitHub run summary, for #62

### CODE REVIEW

**Review by gaborbernat — CHANGES_REQUESTED** [review]
Test for negative_squared_cross_mark please

File: src/tox_gh/plugin.py [hunk 1]
```diff
@@ -75,3 +77,13 @@ def tox_before_run_commands(tox_env: ToxEnv) -> None:
...
         print("::endgroup::")
+        write_to_summary(exit_code == Outcome.OK, tox_env.name)
+
+
+def write_to_summary(success: bool, message: str) -> None:
+    """Write a success or failure value to the github step summary if it exists"""
+    if not GITHUB_STEP_SUMMARY:
+        return
+    success_str = ":white_check_mark:" if success else ":negative_squared_cross_mark:"
+    with open(GITHUB_STEP_SUMMARY, "a+") as f:
```
Comment (gaborbernat) [inline src/tox_gh/plugin.py:88]: use pathlib.open instead

Comment (gaborbernat) [reply src/tox_gh/plugin.py:88]: also no single letter variables please

Comment (lengau) [reply src/tox_gh/plugin.py]: All 3 fixed :-)

**Review by gaborbernat — APPROVED** [review]

**Comment by gaborbernat** [comment]
CI failing

**Comment by lengau** [comment]
Thanks for your help and patience @gaborbernat !

**Pull request merged**